Author: Michael Fiedler
Date:   Mon Feb 22 19:27:38 2016 +0100

    Scroll through pages by mouse wheel even if not in best fit mode

diff --git a/libview/ev-view.c b/libview/ev-view.c
index 5ed2d78..0362d3d 100644
--- a/libview/ev-view.c
+++ b/libview/ev-view.c
@@ -4012,7 +4012,7 @@ ev_view_scroll_event (GtkWidget *widget, GdkEventScroll *event)
                state &= ~GDK_SHIFT_MASK;
        }
 
-       if (state == 0 && view->sizing_mode == EV_SIZING_FIT_PAGE && !view->continuous) {
+       if (state == 0 && !view->continuous) {
                switch (event->direction) {
                case GDK_SCROLL_DOWN:
                case GDK_SCROLL_RIGHT:
